src/unix/netbsd.c

```diff
@@ -234,3 +234,25 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
   uv__free(cp_times);
   return 0;
 }
+
+int uv__random_sysctl(void* buf, size_t len) {
+  static int name[] = {CTL_KERN, KERN_ARND};
+  size_t count, req;
+  unsigned char *p = buf;
```

Comment:
Star should lean left here. Can you also separate the declaration and the assignment.